### ISSUE TITLE
Fix warning in jest to for HomePage.test.jsx

### ISSUE THREAD

**Issue by patcon** [GCTC-NTGC/gc-digital-talent#3524]
Still passes, just noise

Workflow run log: https://github.com/GCTC-NTGC/gc-digital-talent/runs/7544327010?check_suite_focus=true#step:7:12

<img width="864" alt="Screen Shot 2022-07-27 at 12 36 51 PM" src="https://user-images.githubusercontent.com/305339/181302056-2c4a554c-cffe-486e-8f04-a3f8c8228ecb.png">

```
PASS src/js/components/home/HomePage.test.tsx
  ● Console

    console.error
      Warning: Received `true` for a non-boolean attribute `external`.
      
      If you want to write it to the DOM, pass a string instead: external="true" or external={value.toString()}.
          at a
          at CardLink (/home/runner/work/gc-digital-talent/gc-digital-talent/frontend/common/src/components/CardLink/CardLink.tsx:34:7)
          at div
          at div
          at HomePage (/home/runner/work/gc-digital-talent/gc-digital-talent/frontend/admin/src/js/components/home/HomePage.tsx:11:23)
          at IntlProvider (/home/runner/work/gc-digital-talent/gc-digital-talent/frontend/node_modules/react-intl/src/components/provider.js:84:47)

      at printWarning (../node_modules/react-dom/cjs/react-dom.development.js:67:30)
      at error (../node_modules/react-dom/cjs/react-dom.development.js:43:5)
      at validateProperty$1 (../node_modules/react-dom/cjs/react-dom.development.js:3521:9)
      at warnUnknownProperties (../node_modules/react-dom/cjs/react-dom.development.js:3559:21)
      at validateProperties$2 (../node_modules/react-dom/cjs/react-dom.development.js:3583:3)
      at validatePropertiesInDevelopment (../node_modules/react-dom/cjs/react-dom.development.js:8765:5)
      at setInitialProperties (../node_modules/react-dom/cjs/react-dom.development.js:9041:5)
      at finalizeInitialChildren (../node_modules/react-dom/cjs/react-dom.development.js:10[20](https://github.com/GCTC-NTGC/gc-digital-talent/runs/7544327010?check_suite_focus=true#step:7:21)1:3)
```

**Comment by vd1992** [comment]
think this isn't exclusive to Jest

![image](https://user-images.githubusercontent.com/40485260/181375438-2fdce811-b707-4ba2-a322-fdaea6f7d1d3.png)

**Comment by vd1992** [comment]
I think @esizer resolved this in #3543 
jest log is clear in https://github.com/GCTC-NTGC/gc-digital-talent/runs/7582653987?check_suite_focus=true

**Comment by patcon** [comment]
Nice! thanks @vd1992
